### PR TITLE
Some addition and correction to the logic nodes docs

### PR DESCRIPTION
**Pull request by xavco63**

### CODE REVIEW

**Comment by lordloki** [comment]
@IzaZed , could you check these modifications?

File: source/manual/logic_nodes/input/gamepad/gamepad_vibrate.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ Right
    Force with which to rotate the weight in the right handle.
 
 Time
-   Vibrate for this many seconds.
+   How many seconds the vibration have to play.
```
Comment (IzaZed) [inline source/manual/logic_nodes/input/gamepad/gamepad_vibrate.rst:30]: typo, should be "vibration has to"

File: source/manual/logic_nodes/input/keyboard/keyboard_key.rst [hunk 2]
```diff
@@ -15,7 +15,8 @@ Parameters
 ++++++++++++++++++++++++++++++
 
 Input Type
-   Input detection mode.
+   Input detection mode. ``Tap`` is activated the first frame that the key is pressed, ``Down`` 
+   is activated while the key is pressed and ``Up`` is activated once the last frame the key is pressed.
```
Comment (IzaZed) [inline source/manual/logic_nodes/input/keyboard/keyboard_key.rst:19]: Up activates on the frame after the key has been released

File: source/manual/logic_nodes/input/mouse/mouse_status.rst [hunk 3]
```diff
@@ -9,26 +9,28 @@
...
-  todo
+  The X position of the cursor from the top left corner of the game window.
 
 Y Position
-   todo
+   The Y position of the cursor from the top left corner of the game window.
 
 X Movement
-   todo
+   The X movement of the cursor on the game window comparing to previous frame position.
```
Comment (IzaZed) [inline source/manual/logic_nodes/input/mouse/mouse_status.rst:30]: typo, should be "compared"

File: source/manual/logic_nodes/input/mouse/mouse_status.rst [hunk 4]
```diff
@@ -9,26 +9,28 @@
...
 
 Y Movement
-   todo
+   The Y movement of the cursor on the game window comparing to previous frame position.
```
Comment (IzaZed) [inline source/manual/logic_nodes/input/mouse/mouse_status.rst:33]: typo, should be "compared"

File: source/manual/logic_nodes/input/mouse/mouse_status.rst [hunk 5]
```diff
@@ -9,26 +9,28 @@
...
 
 Wheel Difference
-   todo
+   *1* if the wheel scroll up and *-1* if the wheel scroll down, *0* else.
```
Comment (IzaZed) [inline source/manual/logic_nodes/input/mouse/mouse_status.rst:36]: *1* if the wheel is scrolled up and *-1* if the wheel is scrolled down, else *0*.

**Review by IzaZed — CHANGES_REQUESTED** [review]

**Review by IzaZed — APPROVED** [review]

**Pull request merged**